qiskit/transpiler/preset_passmanagers/level2.py

```diff
@@ -156,14 +152,22 @@ def _vf2_match_not_found(property_set):
                 )
             ),
         )
-    elif routing_method == "none":
-        routing_pass = Error(
-            msg="No routing method selected, but circuit is not routed to device. "
-            "CheckMap Error: {check_map_msg}",
-            action="raise",
+        vf2_call_limit = None
+        if pass_manager_config.layout_method is None and pass_manager_config.initial_layout is None:
+            vf2_call_limit = int(5e6)  # Set call limit to ~10 sec with retworkx 0.10.2
+        routing_pm = common.generate_routing_passmanager(
+            routing_pass,
+            target,
+            coupling_map=coupling_map,
+            vf2_call_limit=vf2_call_limit,
+            backend_properties=backend_properties,
+            seed_transpiler=seed_transpiler,
+            use_barrier_before_measurement=not toqm_pass,
```

Comment:
I'm not following this change? The logic looked correct before to me. It's setting the vf2 call limit to either the level 2 value (roughly equivalent to 10sec) if we're not setting an explicit layout method or an initial layout. If we are the vf2 limit should be none so we don't run `VF2PostLayout` because we want to use the user specified layout. (also I think there is a syntax error in the suggestion

Reply:
Not sure what I was intending here. I think I was just moving `vf2_call_limit = int(5e6)` to the parameter at calling time.